### PR TITLE
Fixed Readme and examples #5

### PR DESCRIPTION
**Pull request by sandlbn**
Changes for the configuration file and task manifest plus additionally changes for README.

How to verify it:
- Running snap test 

Testing done:
- Local

### CODE REVIEW

**Comment by kindermoumoute** [comment]
@sandlbn Is it possible to fix "Snap" capitalization and snaptel/snapteld changes in this PR (a different commit if you want)?

**Comment by sandlbn** [comment]
@kindermoumoute Ok done ! Well spotted.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ This builds the plugin in `./build/`
 This collector gathers metrics from mongodb server status command. 
 
 ### Global config
-Global configuration files are described in [snap's documentation](https://github.com/intelsdi-x/snap/blob/master/docs/SNAPD_CONFIGURATION.md). You have to add `"mongodb"` section with following entries:
+Global configuration files are described in [snap's documentation](https://github.com/intelsdi-x/snap/blob/master/docs/snapteld_CONFIGURATION.md). You have to add `"mongodb"` section with following entries:
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ All OSs currently supported by snap:
 
 #### Download the plugin binary:
 
-You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-mongodb/releasess) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for snap packages).
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-mongodb/releasess) page. Download the plugin from the latest release and load it into `snapteld` (`/opt/snap/plugins` is the default location for snap packages).
 
 
 #### To build the plugin binary:
```
Comment (kindermoumoute) [inline README.md]: line 49 & 56: Snap*

File: README.md [hunk 3]
```diff
@@ -82,15 +82,15 @@ Example of running snap mongodb collector and writing data to file.
 Ensure [snap daemon is running](https://github.com/intelsdi-x/snap#running-snap):
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 4]
```diff
@@ -82,15 +82,15 @@ Example of running snap mongodb collector and writing data to file.
...
 * initd: `service snap-telemetry start`
 * systemd: `systemctl start snap-telemetry`
-* command line: `snapd -l 1 -t 0 &`
+* command line: `snapteld -l 1 -t 0 &`
 
 Download and load snap plugins:
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md (hunk 3 above)
Comment (IzabellaRaulin) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (IzabellaRaulin) [inline README.md]: done

**Pull request merged**